candlepin.spec.tmpl

```diff
@@ -44,11 +44,7 @@ BuildRequires: selinux-policy-doc
 
 Requires: java-17 >= 1:17.0.0
 Requires: wget
-%if 0%{?fedora}
-Requires: tomcat
-%else
-Requires: pki-servlet-engine
-%endif
+Requires: tomcat < 10
```

Comment:
After more discussion with Satellite/RHCS/tomcat folks, we decided not to make this update for RHEL8 or existing versions of Satellite. But we will make it for RHEL9.
Can we change this logic to be the following then?:
If (fedora or rhel >= 9)
  Requires: tomcat
else
  Requires: pki-servlet-engine

Reply:
done